File: C/1036 - Fórmula de Bhaskara.c
/* 
   ==UserScript==
 @name         Fórmula de Bhaskara
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1036 - Fórmula de Bhaskara
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>
#include <math.h>

int main()
{
    double A, B, C, R1, R2, Delta;
    
    scanf("%lf%lf%lf", &A, &B, &C);
    
    Delta = pow(B,2)-4*A*C;
    
    if(Delta<0){
        printf("Impossivel calcular\n");
    }
    else if(A == 0){
        printf("Impossivel calcular\n");
    }
    else{
    R1 = (-(B) + sqrt(Delta))/(2*A);
    R2 = (-(B) - sqrt(Delta))/(2*A);
    
    printf("R1 = %.5lf\n", R1);
    printf("R2 = %.5lf\n", R2);
    }

    return 0;
}

File: Java/1036 - Fórmula de Bhaskara.java
/* 
   ==UserScript==
 @name         Fórmula de Bhaskara
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1036 - Fórmula de Bhaskara
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        // Scanner
        Scanner scan = new Scanner(System.in);
        
        double A = scan.nextDouble();
        double B = scan.nextDouble();
        double C = scan.nextDouble();
        
        double delta = Math.pow(B, 2) - (4*A*C);
        
        if(A==0 || delta<0){
            System.out.println("Impossivel calcular");
            return;
        }
        
        delta = Math.sqrt(delta);
        
        double R1 = (-(B) + delta) / (2*A);
        double R2 = (-(B) - delta) / (2*A);
        
        System.out.println(String.format("R1 = %.5f", R1));
        System.out.println(String.format("R2 = %.5f", R2));
        
    }
 
}
